Nodejs/Product/Nodejs/TestFrameworks/Tape/tape.js

```diff
@@ -0,0 +1,75 @@
+var fs = require('fs');
+var path = require('path');
+
+function find_tests(testFileList, discoverResultFile, projectFolder) {
+    var test = findTape(projectFolder);
+    if (test === null) {
+        console.error("NTVS_ERROR:", "Couldn't find 'tape' module relative to", projectFolder);
+        return;
+    }
+    
+    var harness = test.getHarness({ exit: false });
+    var tests = harness["_tests"];
+
+    var count = 0;
+    var testList = [];
+    testFileList.split(';').forEach(function (testFile) {
+
+        var testCases;
+        process.chdir(path.dirname(testFile));
```

Comment:
I did the same as the mocha.js framework here. It is related to Fix #8 - "test explorer doesn't run mocha tests with fs.readFileSync and possibly more". This could be an issue with Tape style tests as well.